pxf/Makefile

```diff
@@ -38,6 +38,10 @@ ifneq "$(VENDOR)" ""
     BUILD_PARAMS+= -Pvendor="$(VENDOR)"
 endif
 
+ifneq "$(PXF_VERSION)" ""
```

Comment:
Not really. We should still be able to run simple dev builds which don't require PXF version